NewForce-at-Mountwest/bangazon-api-farfalle
# Training program controller and unit testing

# Description

Added a controller for training program and unit testing to test get, post, put, and delete methods. A test is also included to check that all get results are in the future if the ?completed=false query string is used.


## Related Ticket(s)


## Steps to Test
Fetch the branch from github and open in visual studio. Click the test button on the top toolbar, then run all tests. All tests should succeed. To test the completed query string, run the api then go to the url https://localhost:5001/api/trainingprogram?completed=false. The resulting list should only show programs that will end in the future. Post, put, and delete methods can be tested manually in postman.


# Checklist:
